frontend/src/App.tsx

```diff
@@ -273,7 +273,9 @@ export const App: React.FC<AppProps> = ({
   // HOTKEYS
   useHotkey("global.runStale", runStaleCells);
   useHotkey("global.save", saveOrNameNotebook);
-  useHotkey("global.interrupt", sendInterrupt);
+  useHotkey("global.interrupt", () => {
```

Comment:
Just curious, why make this change?

Reply:
`sendInterrupt` returns a Promise which messed with the typing changes I made to `useHotkey`